src/components/Header.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import './Header.css';
+import './scss/Header.scss';
 
 const Header = () => (
   <div className="header-container">
-    <h2>Book Store</h2>
+    <h2 className="main-header">Bookstore CMS</h2>
     <nav className="nav-bar">
       <Link className="list-1" to="/">BOOKS </Link>
-      <Link to="/Categories">CATEGORIES</Link>
+      <Link className="list-1" to="/Categories">CATEGORIES</Link>
     </nav>
```

Comment:
- [ ] Based on the requirements for this milestone, styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Kindly make sure that the active navbar button has brighter color and non-active greyish

Your design  | Zeplin design
------------- | -------------
<img width="322" alt="Screenshot 2022-07-14 at 18 13 04" src="https://user-images.githubusercontent.com/77038610/179003098-af4071de-37bb-4eab-9a25-484cb1cf3edc.png">  | <img width="377" alt="Screenshot 2022-07-14 at 18 12 42" src="https://user-images.githubusercontent.com/77038610/179003005-cc1baadb-217b-40f7-8107-252f497c7d98.png">